src/components/Header.tsx

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+
+export const Header: React.FC = () => {
+  return (
```

Comment:
when you don't have any logic inside components and just return jsx then you can delete the word `return` 

and just return jsx in round brackets